omod/src/main/webapp/resources/htmlforms/003-HMIS_HIVCare_ARTCard-ClinicalAssessmentPage.xml

```diff
@@ -0,0 +1,1531 @@
+                    }
+
+                    if (!patient_regimen.includes(regimen_encounter_page_value)) {
+                        jq().toastmessage('showNoticeToast', "The regimen" + regimen_encounter_page_text + "you have selected is different from the current regimen in the Summary page. Please update the summary page with a switch/substitution to the selected regimen");
+                    }
+
+                    if (currentRegimen !== jq("#90315").find("select").find(":selected").val()) {
+                        jq("#duration-on-current-regimen input").val(0);
+                    }else {
+                        jq("#duration-on-current-regimen input").val(durationOnCurrentRegimen);
+                    }
+                    stabilityCheck();
+                });
+                /* End of Validation */
+
+
+                jq("#90203").change(function () {
+                    clinicStaging = jq(this).find(":selected").val();
+                    stabilityCheck();
+                });
+
+                disable_enable_fields(["edd", "emtct", "age-in-months", "anc_no", "district-tb-no", "tb-start-date", "tb-stop-date",
+                                    "no-tpt-pills", "no-b6-pills", "no-of-dapsone-pills", "no-of-dapsone-pills-days",
+                                    "no-of-fluconazole-pills", "no-of-fluconazole-pills-days", "fluconazole-stop-date",
+                                    "viral-load-status", "patient-categorization", "viral-load-results-qualitative",
+                                    "reason_for_next_appointment", "other_reason_for_next_appointment", "other-regimen",
+                                    "no-of-art-pills", "no-of-art-pills-days", "viral-load-results-copies",
+                                    "family-planning-methods"], true);
+
+                jq("#returndate").change(function () {
+                    if (jq("#returndate").find("input[type=hidden]").val() != "") {
+                        disable_enable_fields(["reason_for_next_appointment"], false);
+                    } else {
+                        disable_enable_fields(["reason_for_next_appointment", "other_reason_for_next_appointment"], true);
+                    }
+                });
+
+                jq("#reason_for_next_appointment").change(function () {
+                    if (jq(this).find("select").find(":selected").val() == "90002") {
+                        disable_enable_fields("other_reason_for_next_appointment", false);
+                    } else {
+                        disable_enable_fields("other_reason_for_next_appointment", true);
+                    }
+                });
+
+                jq(".pregnancy").change(function () {
+                    if (jq(this).find('#pregnant').find(':checkbox').prop('checked')) {
+                        disable_enable_fields(["edd"], false);
```

Comment:
@semujju change this function to enableField and disablFields I am sure it already exists from aijar this one is confusing

Reply:
I think this one is straight forward, look at its actual definition in the js file. You pass it a true parameter along with the array of fields you want to enable and a false parameter if u need to disable fields.

Reply:
done